images/doppler-kubernetes-operator/config/template.apko.yaml

```diff
@@ -0,0 +1,16 @@
+contents:
+  packages:
+    - doppler-kubernetes-operator
```

Comment:
you already defined extra_packages for this package, so you don't need to add it here as well, you can remove this one here.